rasa/core/training/interactive.py

```diff
@@ -1130,11 +1150,23 @@ async def _validate_action(
 
     Returns `True` if the prediction is correct, `False` otherwise."""
 
-    question = questionary.confirm(f"The bot wants to run '{action_name}', correct?")
+    if action_name == ACTION_UNLIKELY_INTENT_NAME:
+        question = questionary.confirm(
+            f"The bot wants to run '{action_name}' "
+            f"to indicate that the last user message was unexpected "
+            f"at this point in the conversation. "
+            f"Check out UnexpecTEDIntentPolicy "
+            f"({DOCS_URL_POLICIES}#unexpected-intent-policy) "
+            f"to learn more. Is that correct?"
+        )
+    else:
+        question = questionary.confirm(
+            f"The bot wants to run '{action_name}', correct?"
+        )
 
     is_correct = await _ask_questions(question, conversation_id, endpoint)
 
-    if not is_correct:
+    if not is_correct and action_name != ACTION_UNLIKELY_INTENT_NAME:
```

Comment:
When I train moodbot with UnexpecTEDIntentPolicy and run interactive learning, then start the conversation with "No" and say `no` to use `action_unlikely_intent`, I get into an infinite loop of Rasa asking `The bot wants to run 'action_default_fallback', correct?` that repeats until I choose `yes`. Not sure if this is just an edge case, though.

Reply:
Yes, it's a bug on `main` as well reported [here](https://github.com/RasaHQ/rasa/issues/9080).